frontend/src/App.vue

```diff
@@ -215,6 +217,9 @@ export default {
               question.answer = question.default;
             }
           }
+          if (questionIndex !== changedQuestionIndex && question.type === "password") {
+            question.answer = undefined;
+          }
```

Comment:
place it at end after validate' if.

          if (question.type === "password" && questionIndex !== changedQuestionIndex) {
            question.answer = undefined;
          }